src/components/list/list.scss

```diff
@@ -16,63 +17,161 @@ $item-padding-top: 0px !default;
 $item-padding-right: 0px !default;
 $item-padding-left: 0px !default;
 $item-padding-bottom: 0px !default;
+$list-item-padding-vertical: 0px !default;
+$list-item-padding-horizontal: $baseline-grid * 2 !default;
+$list-item-primary-width: $baseline-grid * 7 !default;
+$list-item-primary-avatar-width: $baseline-grid * 5 !default;
+$list-item-primary-icon-width: $baseline-grid * 3 !default;
+$list-item-secondary-left-margin: $baseline-grid * 2 !default;
+$list-item-text-padding-top: $baseline-grid * 2 !default;
+$list-item-text-padding-bottom: $baseline-grid * 2.5 !default;
 
 md-list {
+  display: block;
   padding: $list-padding-top $list-padding-right $list-padding-bottom $list-padding-left;
+
+  .md-subheader {
+    font-weight: 500;
+    font-size: 0.875em;
```

Comment:
Should this use a variable? Also, should we start using `rem` units for fonts? That's what the new Typography stylesheet uses.